src/visualization/Viewer.js

```diff
@@ -29,7 +29,7 @@ ROS3D.Viewer = function(options) {
   var intensity = options.intensity || 0.66;
   var near = options.near || 0.01;
   var far = options.far || 1000;
-  var cameraPosition = options.cameraPose || {
+  var cameraPosition = options.cameraPosition || {
```

Comment:
That makes much more sense, sorry I am still new to collaborating in projects, but I am trying :)
Reverted in b6bbf7d